packages/types/src/api/web/app/permission.ts

```diff
@@ -14,3 +14,21 @@ export interface GetResourcePermsResponse {
 export interface GetDependantResourcesResponse {
   resourceByType?: Record<string, number>
 }
+
+export interface AddedPermission {
+  _id: string
+  rev?: string
```

Comment:
This seems strange. They should be both _id & _rev or id & rev

Reply:
Hmm, seems to have been an oversight on my part. I have made it optional and everything passes type checking. Good spot.

Reply:
I meant the `_`. But as far as I see, this was already like this :/